src/services/textChanges.ts

```diff
@@ -348,12 +348,18 @@ namespace ts.textChanges {
             this.replaceRangeWithNodes(sourceFile, createRange(pos), newNodes, options);
         }
 
-        public insertNodeAtTopOfFile(sourceFile: SourceFile, newNode: Statement, blankLineBetween: boolean): void {
+        public insertNodesAtTopOfFile(sourceFile: SourceFile, newNodes: Statement | readonly Node[], blankLineBetween: boolean): void {

```

Comment:
I'm not sure I understand why it's `Statement | Node[]` and not `Statement | Statement[]` or `Node | Node[]`.

Reply:
It looks like the other helpers use two separate functions, e.g. `insertNodeAfter` and `insertNodesAfter`.  Of course there's no reason both of those can't delegate to a shared impl.